imblearn/utils/testing.py

```diff
@@ -14,6 +14,11 @@
 
 from sklearn.base import BaseEstimator
 
+from pytest import warns as _warns
+from contextlib import contextmanager
+from re import compile
```

Comment:
the order will be:

* standard library
* 3rd party library
* own library

then 

move context and re on the top
then pytest 
then scikit-learn
then imblearn